File: test/end-to-end/Point/Point.java
//
package classes;

public class Point {
    public int x;
    public int y;
    protected int square;
    private String name;

    public Point(int x, int y) {
        this.x = x;
        this.y = y;
        this.square = x * y;
        this.name = "";
    }
}
//

File: test/end-to-end/Point/Point.cs
//
namespace Classes
{
    public class Point
    {
        public int X;
        public int Y;
        protected int Square;
        private string name;

        public Point(int x, int y)
        {
            this.X = x;
            this.Y = y;
            this.Square = x * y;
            this.name = "";
        }
    }
}
//
